plasma_framework/contracts/src/exits/payment/PaymentExitGame.sol

```diff
@@ -11,6 +11,9 @@ import "../../vaults/Erc20Vault.sol";
 import "../interfaces/IStateTransitionVerifier.sol";
 import "../../utils/OnlyFromAddress.sol";
 
```

Comment:
https://github.com/omisego/plasma-contracts/blob/e5e9e3d0f70b47e8483ee37cc464ce91da593789/plasma_framework/contracts/src/exits/payment/PaymentExitGame.sol#L53 > Don't what that means, could we rephrase this.

Reply:
oh aw, I would remove this line instead then. I was planning to change the word from `token` to `ercContract` previously in the interface of `IExitProcessor`. This is basically trying the explain how `ercContract == token` here. However, as we decide to stick with current word usage, I think this comment is actually not needed anymore.

see: https://github.com/omisego/plasma-contracts/pull/285/files and https://github.com/omisego/plasma-contracts/pull/277